x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_card/product_card.tsx

```diff
@@ -59,12 +82,9 @@ export const ProductCard: React.FC<IProductCard> = ({ product, image }) => {
               metric: snakeCase(product.ID),
             })
           }
-          data-test-subj={`Launch${upperFirst(product.ID)}Button`}
+          data-test-subj={`${TEST_SUBJ_PREFIX}${upperFirst(product.ID)}Button`}
```

Comment:
Just a thought - I originally left it in as (e.g.) `LaunchAppSearchButton` because our current Cypress tests hook into that `data-test-subj` attr for clicking the button. However, if we're going to change it to a dynamic attr, that presumably breaks the Cypress test. 🤔 

Since we're going to have to rewrite our Cypress tests a decent amount anyway, what are your thoughts on just losing `data-test-subj`/`TEST_SUBJ_PREFIX` altogether for now? IMO, for Jest, checking the button's text for Launch vs Setup is pretty sufficient for asserting the conditional.

That being said, I don't feel super strongly about this, so if you'd rather leave this as-is I'm good with that 🤷

Reply:
Sure happy to remove it.